flight-icons/README.md

```diff
@@ -6,11 +6,12 @@ A package for the icons from `flight`, which can be used in React or Ember apps.
 
 ## Installation and Usage
 
-See [https://flight-hashicorp.vercel.app/](https://flight-hashicorp.vercel.app/) for installation and usage instructions.
```

Comment:
@amyrlam you might have been out when we were discussing this, what is the difference between these two URLs?

Reply:
we can change it, i've been directing ppl to the link on the RHS of the repo homepage so should decide on one?

https://hashicorp.slack.com/archives/C025N5V4PFZ/p1630612266013300

Reply:
but they go to the same place, yes? The thinking was that https://flight-hashicorp.vercel.app/ seemed more production-y than the staging url so it would make sense to start using that one by default moving forward.

Reply:
![image](https://user-images.githubusercontent.com/1372946/133335343-750b6857-8cf1-41c0-b2cc-1cec4f6d2a37.png)

^ updated 

sounds good, i deleted flight-staging as a domain. i'm not sure why vercel gives us the other one automatically yet, but not super impt